network/src/p2p.rs

```diff
@@ -240,6 +240,20 @@ impl ReliableNetwork<PrimaryWorkerMessage> for P2pNetwork {
     }
 }
 
+impl UnreliableNetwork<WorkerSynchronizeMessage> for P2pNetwork {
+    type Response = ();
+    fn unreliable_send(
+        &mut self,
+        peer: NetworkPublicKey,
+        message: &WorkerSynchronizeMessage,
+    ) -> Result<JoinHandle<Result<anemo::Response<()>>>> {
+        let message = message.to_owned();
+        let f =
```

Comment:
Not super familiar with the anemo interface: is thif f fut triggered upon send, upon receive?

Reply:
This is the send side